staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -162,22 +162,58 @@ $paths += "c:\AzureData\CustomDataSetupScript.log"
 Write-Host "Collecting Containerd running containers"
 $res = Get-Command ctr.exe -ErrorAction SilentlyContinue
 if ($res) {
-  Write-Host "Collecting Containerd running containers - containers"
+   Write-Host "Collecting ctr plugin ls"
+  & ctr.exe -n k8s.io plugin ls > "$ENV:TEMP\$timeStamp-containerd-plugin.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-plugin.txt"
+
+  Write-Host "Collecting ctr containers"
   & ctr.exe -n k8s.io c ls > "$ENV:TEMP\$timeStamp-containerd-containers.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-containers.txt"
 
-  Write-Host "Collecting Containerd running containers - tasks"
+  Write-Host "Collecting ctr tasks"
```

Comment:
Could you help to put all ctr logs into a folder in your PR? Including existing ctr logs, will make the result clearer, thanks for this improvement.

![image](https://github.com/Azure/AgentBaker/assets/62196586/7918f94e-0039-45cc-9efe-0f511480cf3f)